usage/gpio-plus-and-raspi2/README.md

```diff
@@ -1,89 +0,0 @@
-# GPIO: Models A+, B+, Raspberry Pi 2 B and Raspberry Pi 3 B
```

Comment:
Rather than deleting this page entirely, perhaps it should mention that it's no longer used, and simply contain a link to the 'other' gpio page, so that any external links to it don't suddenly break? \*shrug\*